src/command.c

```diff
@@ -978,9 +992,10 @@ void dv_run_ipu_command(struct dmp_cmb *cmb, void *bar_logical)
 	uint32_t offset;
 	uint32_t val;
 
-	cmb_node = list_first_entry(&cmb->cmb_list, struct dmp_cmb_list_entry, list_node);
+	cmb_node = list_first_entry(&cmb->cmb_list,
+			struct dmp_cmb_list_entry, list_node);
```

Comment:
misalignment, either break line just after the bracket or align next line just after the bracket